taverna-scufl2-cwl/workflow.wfbundle

```diff
@@ -0,0 +1,10 @@
+WorkflowBundle 'bundle1'
```

Comment:
This file has extension .wfbundle but is in text format. Also can we move these to src/test/resources or whatever the files are for?


The file /workflow.vnd.taverna.scufl2.structure.txt` has a weird filename and is not even a text file. Remove.